cluster/addons/cluster-monitoring/heapster-rbac.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: system:heapster-cluster-role
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
```

Comment:
It doesn't. Didn't realize that one was there. I'll drop this one and refer to the existing one.